raiden-dapp/src/store/notifications/mutations.ts

```diff
@@ -34,6 +35,7 @@ export const mutations: MutationTree<NotificationsState> = {
       received: new Date(),
       display,
       context: payload.context ?? NotificationContext.NONE,
+      importance: payload.importance ?? NotificationImportance.LOW,
```

Comment:
I'm still not sure why we have these importance levels?

Reply:
So far this is only used to show highly important notifications with the snackbar stuff.